### PR TITLE
Cleanup and reformat some files

### PR DESCRIPTION
**Pull request by sveneberth**
* remove first utf8-encoding line
* Capitalize classes
* refactor code, optimize imports
* remove obsolete core.run() call (the method no longer exists, replaced
  by assigning app)

### CODE REVIEW

File: deploy/skeletons/example.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#-*- coding: utf-8 -*-
-from viur.core.skeleton import Skeleton
 from viur.core.bones import *
+from viur.core.skeleton import Skeleton
+
 
-class exampleSkel(Skeleton):
+class ExampleSkel(Skeleton):
```
Comment (phorward) [inline deploy/skeletons/example.py:5]: OK in most cases in our projects, `exampleSkel` is standard. This is ok for me, but then we should update all our projects.

File: deploy/main.py [hunk 2]
```diff
@@ -90,6 +89,3 @@
 
 #core.setDefaultLanguage("en") #set default language!
 app = core.setup(modules, render)
-
-if __name__ == "__main__":
```
Comment (phorward) [inline deploy/main.py:94]: Are you sure this will work?

File: deploy/vi_plugins/gen-files-json.py [hunk 3]
```diff
@@ -1,21 +1,19 @@
 #!/usr/bin/env python3
```
Comment (phorward) [inline deploy/vi_plugins/gen-files-json.py:1]: The entire file can be removed, as it is part of flare now.

File: get-pyodide.py [hunk 4]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python3
```
Comment (phorward) [inline get-pyodide.py:1]: File can also be removed. It is now part of flare.

**Review by phorward — CHANGES_REQUESTED** [review]
Hello @sveneberth thank you for your pull request. This looks mostly fine to me, please check out the comments where I'm not sure about.

File: deploy/skeletons/example.py (hunk 1 above)
Comment (phorward) [reply deploy/skeletons/example.py:5]: We should discuss this in Mousebrandround next monday. Are you available?

Comment (sveneberth) [reply deploy/skeletons/example.py:5]: I think that's a de facto standard naming convention. Even in PEP 8. I think we should follow it.

But, we can talk about it...

File: deploy/main.py (hunk 2 above)
Comment (sveneberth) [reply deploy/main.py:94]: Yes, `print(__name__)` prints `main`, so this branch will never be entered. And there is no `run`-method in the core

File: deploy/skeletons/example.py (hunk 1 above)
Comment (phorward) [reply deploy/skeletons/example.py:5]: Yeah, and standards are absolutely welcome. We should just talk about it.

File: deploy/main.py (hunk 2 above)
Comment (phorward) [reply deploy/main.py:94]: Umm okay.. I didn't know about that. Then its okay.

**Review by phorward — APPROVED** [review]
OK I'm fine with this now. Let's talk about the Class naming convention on monday.

**Pull request merged**